assets/js/hooks/Chat.js

```diff
@@ -105,6 +105,11 @@ export default {
             let offendingChatMessage = chatMessages.querySelectorAll(`[data-user-id=${CSS.escape(offendingUserID)}]`);
             // Have to hide them otherwise the tooltip gets stuck on removing the element
             offendingChatMessage.forEach(e => e.hidden = true);
+        });
+
+        this.handleEvent("remove_deleted_message", (e) => {
+            let offendingMessageID = e["message_id"];
+            document.getElementById(offendingMessageID).hidden = true;
```

Comment:
```suggestion
            document.getElementById(e["message_id"]).hidden = true;
```

Since we do not do anything else with offendingMessageID